src/main/java/frc/team2412/robot/util/autonomous/AutonomousChooser.java

```diff
@@ -122,26 +125,31 @@ private static SequentialCommandGroup getLineAutoCommand(Subsystems subsystems,
         // Replace with individual testing commands
         ONE_BALL(
                 (subsystems, trajectories) -> new OneBallAutoCommand(subsystems.indexSubsystem,
+
```

Comment:
Nit: Please remove empty lines (Here, around `SQUARE_PATH`, and in `INTAKE_SHOOTER`).

Reply:
Left the blank line after `INTAKE_SHOOTER` since that's the end of the enum definition block. But got rid of the rest.